app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_person = [
+        Person(person["name"], person["age"]) for person in people
+    ]
+    for name, person in Person.people.items():
+        for info in people:
```

Comment:
loop in loop.. Isn't it too complicated?

Reply:
U may suggest anything else?